lang/libcxx/Portfile

```diff
@@ -102,9 +110,16 @@ if {${os.major} < 11 || [variant_isset replacemnt_libcxx]} {
     compiler.blacklist *gcc* {clang < 500}
 
     # clang 3.5 and newer are conditionally blacklisted to prevent dependency cycles
-    foreach ver {3.5 3.6 3.7 3.8 3.9 4.0 5.0 6.0 7.0 8.0 9.0 devel} {
-        if {![file exists ${prefix}/bin/clang-mp-${ver}]} {
-            compiler.blacklist-append macports-clang-${ver}
+    foreach clang_c ${compiler.fallback} {
+        if {[string match macports-clang-* $clang_c]} {
+            # Extract version number from macports clang compiler name
+            set clang_v [lindex [split $clang_c -] 2]
+            if { ${clang_v} >= 3.5 } {
+                if {![file exists ${prefix}/bin/clang-mp-${clang_v}]} {
+                    ui_debug "Adding ${clang_c} to compiler blacklist"
+                    compiler.blacklist-append ${clang_c}
```

Comment:
It would be a lot more straightforward to just choose a compiler deterministically like the clang ports do. Either the system clang is new enough or it can bootstrap with clang-3.4. The emulated_tls variant complicates that somewhat but bootstrapping that properly is another problem.

Reply:
I don't necessarily disagree, but its was by design not my intention to change the logic in what the port does here, simply to remove the hardcoded clang version list. Other changes like you describe should be the focus of a future PR, not this one.